docs/scan-source.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: page
+title: Project Source Scanning
+parent: Usage
+permalink: /usage/scan-source
+nav_order: 2
+---
+
+# Source Scanning
+
+OSV-Scanner can be used to scan your project source and lockfiles to find vulnerabilities in your dependencies.
+
+```bash
+osv-scanner source scan <flags> [paths...]
```

Comment:
this should be `scan source` right? there's a few references in this page that needs to be fixed.

Reply:
Ah yes of course, fixed now.